BraintreeCore/src/main/java/com/braintreepayments/api/core/ConfigurationException.kt

```diff
@@ -5,7 +5,8 @@ import androidx.annotation.RestrictTo
 /**
  * Error class thrown when a configuration value is invalid
  */
-class ConfigurationException @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) @JvmOverloads constructor(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class ConfigurationException @JvmOverloads constructor(
```

Comment:
I believe merchants reference this class. We should move the RestrictTo annotation to the constructor.

Reply:
Updated!